client/app.jsx

```diff
@@ -1,8 +1,67 @@
 import React from 'react';
-import Home from './pages/home';
+// import Home from './pages/home';
 
 export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      listName: '',
+      userId: 1
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleCancel = this.handleCancel.bind(this);
+  }
+
+  handleChange(event) {
+    const { name, value } = event.target;
+    this.setState({ [name]: value });
+  }
+
+  handleCancel(event) {
+    event.preventDefault();
+    this.setState({ listName: '' });
+  }
+
+  handleSubmit() {
+    event.preventDefault();
+    const req = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        listName: this.state.listName,
+        userId: this.state.userId
+      })
+    };
+    fetch('/api/lists/new-list', req)
+      .then(res => res.json())
+      .then(this.setState({ listName: '' }));
```

Comment:
`.then()` takes a function definition as its argument, but here you are passing a function call.